bibtex-actions.el

```diff
@@ -519,8 +580,11 @@ With prefix ARG, rebuild the cache before offering candidates.
 The bibtex key found by `bibtex-completion-key-at-point' is used
 as the initial input."
   (interactive "P")
-  (funcall bibtex-actions-default-action
-           (bibtex-actions-read :initial 'point :rebuild-cache arg)))
+  (if (and (fboundp 'embark-act)
+           (bibtex-actions-citation-key-at-point))
+      (embark-act)
+    (funcall bibtex-actions-default-action
+             (bibtex-actions-read :initial 'point :rebuild-cache arg))))
```

Comment:
Given the conversation, do we still need/want the initial-value here? Couldn't we just pass the keys to `'bibtex-actions-default-action`?

More consistent, etc.

Reply:
This function has been changed as suggested.